.github/workflows/ci.yml

```
@@ -0,0 +1,34 @@
+name: Code quality check
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)